src/pages/docs/v2/getting-started.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Getting Started With Your ShipEngine Connect Application
+description: Learn how to begin developing your ShipEngine Connect application
+tags:
+  - ShipEngine Connect
+---
+
+Getting Started
+================
+
+In order to add your service to ShipEngine Connect and make it available within our suite of e-commerce applications,
+you'll need to create a JavaScript application.
+
+Our documentation assumes that you understand:
+* Basic programming concepts
+* How to use a terminal on your computer
+* [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)/[NodeJS](https://nodejs.org/)
+* [TypeScript](https://www.typescriptlang.org/) (_optional - you can use plain JavaScript_)
+
+After upgrading your @shipengine/connect to at least **1.12.0** using the `npm install -g @shipengine/connect`, you now have access to a new flag for the init command `connect init --beta`.
```

Comment:
Since we are going to merge and deploy the new CLI lets update this to say 2.0.0 and the command can just be connect init

Reply:
Ah good catch, fixed